sql/information_schema/information_schema.go

```diff
@@ -1082,7 +1082,7 @@ func processListRowIter(ctx *Context, c Catalog) (RowIter, error) {
 			proc.User,                    // user
 			ctx.Session.Client().Address, // host
 			db,                           // db
```

Comment:
Pretty sure this is wrong too: it's going to always return the current DB, not the one the process is connected to

Fix plz

Reply:
Also check out the `show processlist` implementation, I'm not positive that field is correct there either

Reply:
You are right, we actually never set the `Database` var; just added the fix for this.